app/src/androidTest/java/org/mozilla/focus/screenshots/TVScreenshots.java

```diff
@@ -11,40 +11,40 @@
 import android.preference.PreferenceManager;
```

Comment:
Can we also add test to check for "page not found" error here?  i.e. going to invalid url like "firefox:aaa" would open the dialog.

Reply:
There's many errors, I don't know how important these are they seem low priority https://github.com/mozilla-mobile/firefox-tv/blob/master/app/src/main/res/values/strings.xml

Reply:
Right, this reminds me, supposedly it is using the focus string for error message, and if so we won't need to check here again.